types/react-native/index.d.ts

```diff
@@ -5435,8 +5435,9 @@ export class TouchableNativeFeedback extends TouchableNativeFeedbackBase {
      *
      * @param color The ripple color
      * @param borderless If the ripple can render outside it's bounds
+     * @param radius The radius of ripple effect
      */
-    static Ripple(color: ColorValue, borderless?: boolean): RippleBackgroundPropType;
+    static Ripple(color: ColorValue, borderless?: boolean, radius?: number): RippleBackgroundPropType;
```

Comment:
It seems like `borderless` is not optional anymore, can we fix that? Also it might be worth it to add a simple test for it

Reply:
I added tests and made the `borderless` not optional.